### ISSUE TITLE
rpma_log_set_function.3 contains an invalid macro `..`

### ISSUE THREAD

**Issue by kilobyte** [pmem/rpma#564]
```
W: librpma-dev: groff-message usr/share/man/man3/rpma_log_set_function.3.gz 44: warning: macro '..' not defined
```

And indeed, this is not a benign warning: the man page contains an ugly non-indented bold (in "man") or blue-background (in "mc") `...);` on the left side of the screen.

Apparently, *src2man* fails to cope with lines starting with a dot.  Not sure how to fix this problem — perhaps moving the dots elsewhere or using a non-ASCII dot would be a good enough workaround?

**Comment by ldorau** [comment]
@kilobyte what Linux distribution and version this issue occurs on? I cannot reproduce it.

**Comment by ldorau** [comment]
@kilobyte It does not reproduce on the current (docker image) Debian Experimental either

**Comment by kilobyte** [comment]
Reproduces for me on Debian unstable and testing; can't test Fedora as src2man fails there.

You can either view the manpage, or run:
```
man --warnings >/dev/null ./rpma_log_set_function.3
```

**Comment by ldorau** [comment]
> Reproduces for me on Debian unstable and testing; can't test Fedora as src2man fails there.
> 
> You can either view the manpage, or run:
> 
> ```
> man --warnings >/dev/null ./rpma_log_set_function.3
> ```

Thanks @kilobyte !

**Comment by ldorau** [comment]
@kilobyte please test the fix: #584

**Comment by ldorau** [comment]
Fixed by #584
